.github/release-drafter.yml

```
@@ -60,7 +60,7 @@ autolabeler:
       - "/^(fix).*/i"
   - label: "infrastructure"
     title:
-      - "/^(infrastructure).*/i"
+      - "/^(infra).*/i"
```

```suggestion
      - "/^(infrastructure|infra).*/i"
```

Up for keeping both just in case?